chromium_src/chrome/browser/upgrade_detector/get_installed_version_mac.mm

```diff
@@ -33,7 +33,7 @@ InstalledAndCriticalVersion GetInstalledVersionSynchronous() {
 #undef GetInstalledVersion
 
 void GetInstalledVersion(InstalledVersionCallback callback) {
-  if (brave::ShouldUseOmaha4()) {
+  if (brave::updater::ShouldUseOmaha4()) {
     GetInstalledVersion_ChromiumImpl(std::move(callback));
   } else {
     std::move(callback).Run(GetInstalledVersionSynchronous());
```

Comment:
In a follow-up we should fix this and implement `CurrentlyInstalledVersion` instead of overriding this, but it's not clear why we need this override in the first place. Is keystone returning something different from what is in the Info.plist? That seems wrong.

Reply:
or I guess maybe it's returning the truncated brave version with the chromium major version? That's something else we really need to address at some point, but either way it's better to handle this with an override of `CurrentlyInstalledVersion`